From 6a84c04cabb2a8a8572faa39d2f3645ed3cc063a Mon Sep 17 00:00:00 2001
From: rifux <contact@rifux.dev>
Subject: [PATCH] socrates: Remove device_framework_matrix.xml from
 BoardConfig.mk

Not needed by this ROM & causes FAILED: ninja: ... error

Signed-off-by: rifux <contact@rifux.dev>
---
 BoardConfig.mk | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index d0548b1..96fcf63 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -195,8 +195,7 @@ TARGET_QTI_VIBRATOR_USE_EFFECT_STREAM := true
 # VINTF
 DEVICE_MATRIX_FILE := $(DEVICE_PATH)/configs/vintf/compatibility_matrix.xml
 DEVICE_FRAMEWORK_COMPATIBILITY_MATRIX_FILE := \
-    $(DEVICE_PATH)/configs/vintf/framework_matrix.xml \
-    vendor/voltage/config/device_framework_matrix.xml
+    $(DEVICE_PATH)/configs/vintf/framework_matrix.xml
 DEVICE_MANIFEST_FILE := $(DEVICE_PATH)/configs/vintf/manifest.xml
 
 # Vendor security patch
-- 
2.47.0

